db/migrate/20200527004935_create_users.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUsers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :address
+      t.string :city
+      t.string :state
+      t.string :zip
+      t.string :email
+      t.string :password_digest
+      t.integer :role
+
```

Comment:
THIS IS SO GOOOD

Reply:
![](https://media.giphy.com/media/U6pavBhRsbNbPzrwWg/giphy.gif)

Reply:
![](https://media.giphy.com/media/l0HeiaW8q9B6tqoHS/giphy.gif)